tests/resources/stitchedUpTechRecords.json

```diff
@@ -0,0 +1,532 @@
+{
```

Comment:
Having them in a seperate file to the technical-records.json means we woulnd't be able to seed these. Is that intended?

Reply:
Yes, this file is just used for checking the result of an integration test.